index.html

```diff
@@ -661,24 +734,6 @@
                     mxEvent.consume(evt);
                 };
 
-                // Returns a shorter label if the cell is collapsed and no
-                // label for expanded groups
-                graph.getLabel = function(cell) {
```

Comment:
It didn't actually, everything works without it. This piece of code is actually meant for uncollapsing collapsed mxCells - something which has no use in our functionality.